### PR TITLE
[FR] Update `Client/Release stream/Lazer/File storage`

### PR DESCRIPTION
**Pull request by Pisapou**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 1]
```diff
@@ -38,15 +33,21 @@ Pour le moment, la procédure correcte pour manuellement modifier les skins ou l
...
 
 À noter que si l'utilisation du disque est vérifiée depuis la fenêtre `Propriétés` dans l'Explorateur ou d'une quelconque autre méthode similaire, il semblera que les dossiers `Songs` et `files` utilisent de l'espace disque, ce qui pourrait induire en erreur les utilisateurs à penser que les données sont stockées deux fois. C'est un artéfact de la détermination de la taille du fichier, et les vraies données stockées sur le disque ne sont **pas** dupliquées. Cela peut être vérifié en comparant la valeur totale de l'espace disponible sur le disque entier avant et après la migration.
 
-Comme les liens physiques sont une fonctionnalité au niveau du système de fichiers, il est obligatoire que les deux fichiers liés physiquement se trouvent sur un même disque, et que ce disque soit formaté avec le format NTFS.
+#### Supported systems
```
Comment (Biokax) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:44]:
```suggestion
#### Systèmes supportés
```

Comment (Biokax) [reply wiki/Client/Release_stream/Lazer/File_storage/fr.md:44]: petit oubli

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 2]
```diff
@@ -38,15 +33,21 @@ Pour le moment, la procédure correcte pour manuellement modifier les skins ou l
...
 
 À noter que si l'utilisation du disque est vérifiée depuis la fenêtre `Propriétés` dans l'Explorateur ou d'une quelconque autre méthode similaire, il semblera que les dossiers `Songs` et `files` utilisent de l'espace disque, ce qui pourrait induire en erreur les utilisateurs à penser que les données sont stockées deux fois. C'est un artéfact de la détermination de la taille du fichier, et les vraies données stockées sur le disque ne sont **pas** dupliquées. Cela peut être vérifié en comparant la valeur totale de l'espace disponible sur le disque entier avant et après la migration.
 
-Comme les liens physiques sont une fonctionnalité au niveau du système de fichiers, il est obligatoire que les deux fichiers liés physiquement se trouvent sur un même disque, et que ce disque soit formaté avec le format NTFS.
+#### Systèmes supportés
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:44]:
```suggestion
#### Systèmes pris en charge
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 3]
```diff
@@ -38,15 +33,21 @@ Pour le moment, la procédure correcte pour manuellement modifier les skins ou l
...
+
+Comme les liens physiques sont une fonctionnalité au niveau du système de fichiers, il est obligatoire que les deux fichiers liés physiquement se trouvent sur un même disque.
+
+- **Windows**: Le lecteur doit être formaté en NTFS.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:48]:
```suggestion
- **Windows** : Le lecteur doit être formaté en NTFS.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 4]
```diff
@@ -38,15 +33,21 @@ Pour le moment, la procédure correcte pour manuellement modifier les skins ou l
...
+- **macOS**: Le système de fichiers doit prendre en charge les liens physiques.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:49]:
```suggestion
- **macOS** : Le système de fichiers doit prendre en charge les liens physiques.
```

File: wiki/Client/Release_stream/Lazer/File_storage/fr.md [hunk 5]
```diff
@@ -38,15 +33,21 @@ Pour le moment, la procédure correcte pour manuellement modifier les skins ou l
...
+- **Linux**: Le système de fichiers doit prendre en charge les liens physiques.
```
Comment (Ganondorf-1) [inline wiki/Client/Release_stream/Lazer/File_storage/fr.md:50]:
```suggestion
- **Linux** : Le système de fichiers doit prendre en charge les liens physiques.
```

**Review by Ganondorf-1 — CHANGES_REQUESTED** [review]

**Review by Ganondorf-1 — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**